autoload/airline/extensions/wordcount.vim

```diff
@@ -82,13 +82,15 @@ endfunction
 
 " airline functions {{{1
 " default filetypes:
-let s:filetypes = ['help', 'markdown', 'rst', 'org', 'text', 'asciidoc', 'tex', 'mail']
+let g:airline#extensions#wordcount#filetypes =
+    \ ['help', 'markdown', 'rst', 'org', 'text', 'asciidoc', 'tex', 'mail']
+let s:lastknownfiletypes = g:airline#extensions#wordcount#filetypes
 function! airline#extensions#wordcount#apply(...)
-  let filetypes = get(g:, 'airline#extensions#wordcount#filetypes', s:filetypes)
+  let filetypes = g:airline#extensions#wordcount#filetypes
 
   " Check if filetype needs testing
-  if did_filetype() || filetypes isnot s:filetypes
-    let s:filetypes = filetypes
+  if did_filetype() || filetypes isnot s:lastknownfiletypes
+    let s:lastknownfiletypes = filetypes
```

Comment:
what is lastknownfiletypes?

Reply:
I have to admit I'm not totally sure. I made this change ages ago in my private copy and completely forgot to PR it until I tried moving to a different computer and rediscovered it.